exopy_hqc_legacy/instruments/drivers/visa_tools.py

```diff
@@ -231,6 +231,14 @@ def read_raw(self):
         """
         return self._driver.read_raw()
 
+    def read_bytes(self, n):
```

Comment:
You may want to preserve pyvisa signature (break_on_termchar)

Reply:
Good point, I'll update it